src/test/java/site/onandoff/member/presentation/MemberControllerTest.java

```diff
@@ -0,0 +1,146 @@
+package site.onandoff.member.presentation;
+
+import static org.mockito.BDDMockito.*;
+import static org.springframework.restdocs.mockmvc.MockMvcRestDocumentation.*;
+import static org.springframework.restdocs.operation.preprocess.Preprocessors.*;
+import static org.springframework.restdocs.payload.PayloadDocumentation.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.hibernate.validator.internal.engine.path.PathImpl;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.MediaType;
+import org.springframework.restdocs.payload.JsonFieldType;
+
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.ConstraintViolationException;
+import site.onandoff.RestDocsSupport;
+import site.onandoff.member.application.MemberService;
+import site.onandoff.member.dto.SignUpForm;
+import site.onandoff.member.dto.SignUpSuccessResponse;
+import site.onandoff.member.dto.UniqueSignUpForm;
+
+class MemberControllerTest extends RestDocsSupport {
+
+	private final MemberService memberService = mock(MemberService.class);
+
+	@Override
+	protected Object initController() {
+		return new MemberController(memberService);
+	}
+
+	@Test
+	@DisplayName("회원가입 성공 시, Redirect URL 과 회원이 DB에 저장될때의 pk값을 반환한다.")
+	void signUpSuccess() throws Exception {
+		// given
+		SignUpForm signUpForm = new SignUpForm("ghkdgus29@naver.com", "hyun", "1234567a!");
+
+		given(memberService.signUp(any(UniqueSignUpForm.class)))
+			.willReturn(new SignUpSuccessResponse(1L));
```

Comment:
```suggestion
			.willReturn(new SignUpSuccessResponse(MEMBER_ID));
```
💁🏻 숫자 1이 의미하는 바를 적어두는 건 어떤가요?

Reply:
좋은 생각입니다!
수정하겠습니다!